tensorrt_llm/llmapi/tokenizer.py

```diff
@@ -267,6 +277,417 @@ def hf_decode_incrementally(
             return prev_text + curr_new_text, states
 
 
+# Adapted from:
+# https://github.com/vllm-project/vllm/blob/8e67b2557aae7204c697d7a5c61e00754da465be/vllm/transformers_utils/tokenizers/mistral.py#L166
+class MistralTokenizer(TransformersTokenizer):
```

Comment:
Could we split this implementation to another file?

Reply:
I will do it in another PR. Thank you to comment.